alyssabull/idk
# Feature/initial styles

## What does this PR do?
- Adds initial styles to the application
- Fixes bug that did not remove activities from localStorage
- Adds localStorage to the currentActivity so it is not lost on page refresh
- Makes links (if applicable) functional and open in a new tab

## Where should the reviewer start?
- All `scss` files

## How should this be manually tested?
- Open the app and view all the styles, ensure everything is easy to navigate and read
- Generate a random activity and refresh the page, the activity should remain
- Remove activities from saved activities and ensure 1) a message pops up when there are no activities left 2) refresh the page and ensure all activities stay deleted

## Screenshots
Home Page View
<img width="1301" alt="Screen Shot 2021-01-14 at 8 24 52 PM" src="https://user-images.githubusercontent.com/58453566/104679364-be2d4900-56aa-11eb-95a8-b257bb6965f5.png">

Random Activity View
<img width="1289" alt="Screen Shot 2021-01-14 at 8 54 58 PM" src="https://user-images.githubusercontent.com/58453566/104679401-d8672700-56aa-11eb-8bcb-e3cb4a719281.png">

Saved Activities View
<img width="1287" alt="Screen Shot 2021-01-14 at 8 55 11 PM" src="https://user-images.githubusercontent.com/58453566/104679410-dd2bdb00-56aa-11eb-8372-eab0d5d6d0bb.png">
